test/el_cl_launcher_test.star

```diff
@@ -0,0 +1,153 @@
+el_cl_launcher = import_module("/src/el_cl_launcher.star")
+input_parser = import_module("/src/package_io/input_parser.star")
+observability = import_module("/src/observability/observability.star")
+ethereum_package_constants = import_module(
+    "github.com/ethpandaops/ethereum-package/src/package_io/constants.star"
+)
+util = import_module("/src/util.star")
+
+
+def test_launch_with_defaults(plan):
+    parsed_input_args = input_parser.input_parser(
+        plan,
+        {
```

Comment:
nit: it might be slightly easier to follow if the test inputs were in variables at the top of the test (instead of being inlined in the function call, or further down)
One step closer to table-testing I guess :)

Reply:
The idea here is to add more of these args - this function only tests using the defaults, that's why I kept it in the function (hence `test_launch_with_defaults`)